temporal-serviceclient/src/main/java/io/temporal/internal/retryer/GrpcMessageTooLargeException.java

```diff
@@ -0,0 +1,7 @@
+package io.temporal.internal.retryer;
+
+public class GrpcMessageTooLargeException extends RuntimeException {
```

Comment:
Is this type really internal? If a user makes a client call with a payload that is to large won't that client call throw this exception?